spec/factories/user.rb

```diff
@@ -15,6 +15,10 @@
     factory :admin do
       is_admin true
     end
+
+    factory :non_admin do
+      is_admin false
```

Comment:
It is purely done for clarity.
Because we would have 
`let(:user) { build:(user) }
let(:user) { build(:admin) }`
Which is a bit confusing compared within the specs than
`let(:user) { build(:admin) }
let(:user) { build(:non-admin) }`